### PR TITLE
[FIX] Geenify

### PR DESCRIPTION
**Pull request by adrienpeiffer**
- [ ] Depends on #116

### CODE REVIEW

**Comment by sbidoul** [comment]
:+1:

**Comment by lmignon** [comment]
:+1: but it seems that a dependency on https://github.com/OCA/account-analytic/pull/44 misses in order to have a successful build in travis

**Comment by adrienpeiffer** [comment]
Rebased

File: hr_timesheet_details_report/report/hr_timesheet_details_report.py [hunk 1]
```diff
@@ -79,8 +79,8 @@ def render_html(self, data=None):
             if single_date not in timesheet_by_dates:
                 timesheet_by_dates[single_date] = [False]
             days.append(single_date)
-        wiz = self.env['hr.timesheet.details.report.wizard'].\
-            browse([data['id']])
+        wiz = self.env['hr.timesheet.details.report.wizard']\
+            .browse([data['id']])
```
Comment (sbidoul) [inline hr_timesheet_details_report/report/hr_timesheet_details_report.py:83]: ça c'est spécial

Comment (adrienpeiffer) [inline hr_timesheet_details_report/report/hr_timesheet_details_report.py:83]: Vieux surtout ... je vais le retirer ...

**Comment by sbidoul** [comment]
Closing old stuff.

**Pull request closed without merging**